tgui/packages/tgui/interfaces/APC.js

```diff
@@ -0,0 +1,190 @@
+            <ProgressBar
+              color="good"
+              value={adjustedCellChange} />
+          </LabeledList.Item>
+          <LabeledList.Item
+            label="Charge Mode"
+            color={chargingStatus.color}
+            buttons={(
+              <Button
+                icon={data.chargeMode ? 'sync' : 'close'}
+                content={data.chargeMode ? 'Auto' : 'Off'}
+                disabled={locked}
+                onClick={() => act('charge')} />
+            )}>
+            [ {chargingStatus.chargingText} ]
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+      <Section title="Power Channels">
+        <LabeledList>
+          {channelArray.map(channel => {
+            const { topicParams } = channel;
+            return (
+              <LabeledList.Item
+                key={channel.title}
+                label={channel.title}
+                buttons={(
+                  <Fragment>
+                    <Box inline mx={2}
+                      color={channel.status >= 2 ? 'good' : 'bad'}>
+                      {channel.status >= 2 ? 'On' : 'Off'}
+                    </Box>
+                    <Button
+                      icon="sync"
+                      content="Auto"
+                      selected={!locked && (
+                        channel.status === 1 || channel.status === 3
+                      )}
+                      disabled={locked}
+                      onClick={() => act('channel', topicParams.auto)} />
+                    <Button
+                      icon="power-off"
+                      content="On"
+                      selected={!locked && channel.status === 2}
+                      disabled={locked}
+                      onClick={() => act('channel', topicParams.on)} />
+                    <Button
+                      icon="times"
+                      content="Off"
+                      selected={!locked && channel.status === 0}
+                      disabled={locked}
+                      onClick={() => act('channel', topicParams.off)} />
+                  </Fragment>
+                )}>
+                {channel.powerLoad}
+              </LabeledList.Item>
+            );
+          })}
+          <LabeledList.Item label="Total Load">
+            <b>{data.totalLoad}</b>
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+      <Section
+        title="Misc"
+        buttons={!!data.siliconUser && (
+          <Fragment>
+            {!!data.malfStatus && (
+              <Button
+                icon={malfStatus.icon}
+                content={malfStatus.content}
+                color="bad"
+                onClick={() => act(malfStatus.action)} />
+            )}
+            <Button
+              icon="lightbulb-o"
+              content="Overload"
+              onClick={() => act('overload')} />
+          </Fragment>
+        )}>
+        <LabeledList.Item
```

Comment:
Nitpick: the margin in this section seems inconsistent with the other sections:
![image](https://user-images.githubusercontent.com/1496804/88035151-fe6b9e80-cb41-11ea-8717-3c317aafabc2.png)